update-solana-dependencies.sh

```diff
@@ -47,4 +47,5 @@ crates=(
 set -x
 for crate in "${crates[@]}"; do
   sed -E -i'' -e "s:(${crate} = \")(=?)${old_solana_ver}\".*:\1\2${solana_ver}\":" "${tomls[@]}"
+  sed -E -i'' -e "s:(${crate} = \{ version = \")(=?)${old_solana_ver}(\".*):\1\2${solana_ver}\3:" "${tomls[@]}"
```

Comment:
hahaha, i wasnt fast enough to sneakily commit my driveby, i wanted to test it first
```
-  sed -E -i'' -e "s:(${crate} = \")(=?)${old_solana_ver}\".*:\1\2${solana_ver}\":" "${tomls[@]}"
+  sed -E -i'' -e "s:(${crate} = )(\{ version = ){0,1}(\")(=?)${old_solana_ver}\".*:\1\2\3\4${solana_ver}\":" "${tomls[@]}"
```

Reply:
Hehe, thanks! looks like we pretty much got to the same place, right?

Reply:
ya its equivalent